### PR TITLE
UpdateService returns collaborators without real name #1455

### LINKED ISSUE #1455: UpdateService returns collaborators without real name

### PR DESCRIPTION
**Pull request by raisfathin**
Fixes #1455

### CODE REVIEW

**Comment by raisfathin** [comment]
Ready for review!

**Comment by raisfathin** [comment]
@dariusf can help me review this? Thanks!

File: src/main/java/github/CollaboratorServiceEx.java [hunk 1]
```diff
@@ -34,10 +32,18 @@ public CollaboratorServiceEx(GitHubClient client) {
...
     @Override
     public List<User> getCollaborators(IRepositoryIdProvider repository) throws IOException {
-        return super.getCollaborators(repository).stream()
+        return getUsersCompleteData(super.getCollaborators(repository));
+    }
+
+    /**
+     * Gets the complete data for every user since most of GitHub APIs do not return users' complete data
+     */
+    public static List<User> getUsersCompleteData(List<User> users) {
```
Comment (dariusf) [inline src/main/java/github/CollaboratorServiceEx.java]: Maybe this could be named `getCompleteUserData` or `getCompleteUser`?

**Comment by dariusf** [comment]
Nice work!

**Comment by raisfathin** [comment]
Made the suggested change!

**Pull request merged**